lighthouse-core/test/test-utils.js

```diff
@@ -238,41 +177,49 @@ async function flushAllTimersAndMicrotasks(ms = 1000) {
  * shouldn't concern themselves about.
  */
 function makeMocksForGatherRunner() {
-  jest.mock('../gather/driver/environment.js', () => ({
+  jest.mock(require.resolve('../gather/driver/environment.js'), () => ({
     getBenchmarkIndex: () => Promise.resolve(150),
```

Comment:
Even though this module is calling `jest.mock`, it seems that in esmodule test files the file is mocked relative to the test file instead. Using `require.resolve` to obtain an absolute path fixes that.